src/instructlab/model/evaluate.py

```diff
@@ -167,37 +177,55 @@ def display_error_rate(error_rate) -> None:
 
 
 def display_branch_eval_summary(
-    improvements: list[tuple[str, float]],
-    regressions: list[tuple[str, float]],
-    no_changes: list[str],
+    benchmark: Benchmark,
+    improvements: list[tuple[str, float, float, float]],
+    regressions: list[tuple[str, float, float, float]],
+    no_changes: list[tuple[str, float]],
     new=None,
 ):
     """takes in results lists from mt_bench_branch benchmark evaluation
     prints out diff between the branches to the user
     """
+    # total score for MT-BENCH-BRANCH
+    total_score = "10.00"
+    if benchmark == Benchmark.MMLU_BRANCH:
+        total_score = "1.00"
+
     if len(improvements) > 0:
         improvements.sort(key=sort_score, reverse=True)
         print("\n### IMPROVEMENTS:")
         for index, improvement in enumerate(improvements):
-            task, delta = improvement
-            print(f"{index+1}. {task} (+{delta})")
+            task, delta, base_score, new_score = improvement
+            base_score = round(base_score, 2)
+            new_score = round(new_score, 2)
+            print(
+                f"{index+1}. {task}: {base_score}/{total_score} -> {new_score}/{total_score} (+{delta})"
```

Comment:
I would vote not to show total score for each entry and only show it with the model at the top.  It makes the output too busy.